SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReplicationGraphLoadBalancingHandler.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialReplicationGraphLoadBalancingHandler.h"
+#include "EngineClasses/SpatialReplicationGraph.h"
+
+FSpatialReplicationGraphLoadBalancingContext::FSpatialReplicationGraphLoadBalancingContext(USpatialNetDriver* InNetDriver, USpatialReplicationGraph* InReplicationGraph, FPrioritizedRepList& InRepList)
+	: NetDriver(InNetDriver)
+	, ReplicationGraph(InReplicationGraph)
+	, ActorsToReplicate(InRepList)
+{
+
+}
+
+
+FSpatialReplicationGraphLoadBalancingContext::FRepListArrayAdaptor FSpatialReplicationGraphLoadBalancingContext::GetActorsBeingReplicated()
+{
+	return FRepListArrayAdaptor(ActorsToReplicate);
+}
+
+void FSpatialReplicationGraphLoadBalancingContext::RemoveAdditionalActor(AActor* Actor)
+{
+
+}
+
+void FSpatialReplicationGraphLoadBalancingContext::AddActorToReplicate(AActor* Actor)
+{
+	ReplicationGraph->ForceNetUpdate(Actor);
```

Comment:
Won't this only get parsed on the next frame? Don't we need to add this actor to PrioritizedReplicationList?